### PR TITLE
Added selecting element with n

### PR DESCRIPTION
**Pull request by andrewmamus**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

**Comment by andrewmamus** [comment]
Many people taught that they can only have access to the elements with class or id

**Comment by RandellDawson** [comment]
Closed and Reopened this PR to attempt to resolve a specific Travis build failure.

File: guide/english/css/css3-nth-child-selector/index.md [hunk 1]
```diff
@@ -49,10 +49,42 @@ For example, the following will match every 3rd anchor element:
     /* CSS goes here */
   }
 ```
+#### Selecting element with n
+The nth child selector saves you from adding class to everything in your website. You can access each element with n being the number of the position of the element.
+```html
+  	<ul class="parent">
```
Comment (co16353sidak) [inline guide/english/css/css3-nth-child-selector/index.md]: Slight formatting issue here:
Align the opening tag `<ul>` with closing `</ul>`

File: guide/english/css/css3-nth-child-selector/index.md [hunk 2]
```diff
@@ -49,10 +49,42 @@ For example, the following will match every 3rd anchor element:
...
+      <li>First<li>
+      <li>Second</li>
+     <li>Third</li>
```
Comment (co16353sidak) [inline guide/english/css/css3-nth-child-selector/index.md]: This `<li>` tag seems out of place with the rest, consider adding a space to make it even

File: guide/english/css/css3-nth-child-selector/index.md [hunk 3]
```diff
@@ -49,10 +49,42 @@ For example, the following will match every 3rd anchor element:
...
+    color:green;
+  }
+  
+    .parent li:nth-child(4){
+    color:black;
+  }
+  
+  .parent li:nth-child(5){
+  color:brown;
+  }
```
Comment (co16353sidak) [inline guide/english/css/css3-nth-child-selector/index.md]: consider adding spaces between property and value, such as:
`color:brown` -> `color: brown` for improved readability

**Review by theNaut1lus — CHANGES_REQUESTED** [review]
@andrewmamus 
Simple typographical changes, consider making them.

/cc: @RandellDawson

**Review by RandellDawson — CHANGES_REQUESTED** [review]
@andrewmamus Please make the suggestions from @co16353sidak for this PR.

Thank you.

**Comment by RandellDawson** [comment]
Hi @andrewmamus 

As there has been no response from your end for a while, we are closing this PR.

Feel free to request this PR to be reopened or create a new PR, if you still wish to make the changes you originally added plus any additional change requests asked of you.  Also, remember to update your fork, so you have the latest version of master in case it has changed significantly.

Happy Coding!

**Pull request closed without merging**